types/react-native/index.d.ts

```diff
@@ -4535,7 +4539,8 @@ export interface SectionListScrollParams {
     viewPosition?: number;
 }
 
-export class SectionList<SectionT = any> extends React.Component<SectionListProps<SectionT>> {
+export class SectionList<ItemT = any, SectionT = DefaultSectionT> extends React.Component<SectionListProps<ItemT,
```

Comment:
This does make it a breaking change, if somebody _had_ specified a type here, no? Having said that, I’m fine with making the change.

Reply:
I'm not sure. As far as I can tell, this generic param was called SectionT but in fact, it is used as ItemT. So, essentially, we're adding a new, second param and simply renaming the first while retaining its semantics and usage.

Reply:
Ah gotcha 👍